content/20.pharmaceuticals.md

```diff
@@ -127,21 +128,51 @@ Therefore, even though the study reported clinical improvement in 68% of the 53
 These trials include [@doi:10.1038/s41422-020-0282-0; @clinicaltrials:NCT04292730; @url:https://www.clinicaltrialsregister.eu/ctr-search/trial/2020-000936-23/FR; @clinicaltrials:NCT04252664; @clinicaltrials:NCT04257656].
 As of October 22, 2020, remdesivir received FDA approval based on three clinical trials [@url:https://www.fda.gov/news-events/press-announcements/fda-approves-first-treatment-covid-19].
 
+However, results suggesting no effect of remdesivir on survival were reported by the WHO Solidarity trial [@doi:10.1056/NEJMoa2023184].
+This large-scale, open-label trial enrolled 11,330 adult in-patients at 405 hospitals in 30 countries around the world [@doi:10.1056/NEJMoa2023184].
+Patients were randomized in equal proportions into four experimental and a control conditions, corresponding to four candidate treatments for COVID-19 and SOC, respectively; no placebo was administered.
+The 2750 patients in the remdesivir group were administered 200 mg intravenously on the first day and 100 mg on each subsequent day until day 10 and assess for in-hospital death (primary endpoint), duration of hospitalization, and progression to mechanical ventilation.
+There were also 2708 control patients who would have been eligible and able to receive remdesivir were they not assigned to the control group.
+A total of 604 patients among these two cohorts deceased during initial hospitalization.
+The rate ratio of death between these two groups was therefore not significant (p = 0.50), suggesting that the administration of remdesivir did not affect survival.
+The two secondary analyses similarly did not find any effect of remdesivir.
+Additionally, the authors compared data from their study with data from three other studies of remdesivir (including [@doi:10.1056/NEJMoa2007764]) stratified by supplemental oxygen status.
+The adjusted rate ratio for death based on this meta-analysis was 0.91.
+These results thus do not support the previous findings that remdesivir reduced median recovery time and mortality risk in COVID-19 patients.
+
+In response to the results Solidarity trial, Gilead, which manufactures remdesivir under the name Veklury^R, released a statement pointing to the fact that the Solidarity trial was not placebo-controlled or double-blind and at the time of the statement had not been peer reviewed [@url:https://www.gilead.com/news-and-press/company-statements/gilead-sciences-statement-on-the-solidarity-trial]; these sentiments have been echoed elsewhere [@doi:10.1093/eurheartj/ehaa934].
+Other critiques of this study have noted that antivirals are not typically targeted at patients with severe illness, and therefore remdesivir could be more beneficial for patients with mild rather than severe case [@url:https://www.sciencemag.org/news/2020/10/very-very-bad-look-remdesivir-first-fda-approved-covid-19-drug; @doi:10.1056/NEJMe2034294].
```

Comment:
```suggestion
Other critiques of this study have noted that antivirals are not typically targeted at patients with severe illness, and therefore remdesivir could be more beneficial for patients with mild rather than severe cases [@url:https://www.sciencemag.org/news/2020/10/very-very-bad-look-remdesivir-first-fda-approved-covid-19-drug; @doi:10.1056/NEJMe2034294].
```